src/monodroid/monodroid.props

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <_CommonDefines>-DJDK_INCLUDE="@(JdkIncludePath->'%(Identity)', ' ')" -DMONO_PATH=$(MonoSourceFullPath) -DSGEN_BRIDGE_VERSION=$(MonoSgenBridgeVersion)</_CommonDefines>
+    <_CommonDefines>-DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DJDK_INCLUDE="@(JdkIncludePath->'%(Identity)', ' ')" -DMONO_PATH=$(MonoSourceFullPath) -DSGEN_BRIDGE_VERSION=$(MonoSgenBridgeVersion)</_CommonDefines>
```

Comment:
What's `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` do?

Reply:
It generates a file called `compile_commands.json` in the temporary directory which contains command lines to compile all the files. This file is used by some editors to implement dependency tracking (e.g. by some [LSP](https://langserver.org/) servers). Really harmless and quick to generate and comes useful for folks not using the "standard" IDEs.